src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Subscales/ActivityCompletionScores/ActivityCompletionScores.tsx

```diff
@@ -83,7 +83,8 @@ export const ActivityCompletionScores = ({
       <BarChart chartData={subscaleScores} />
       {optionText && (
         <Box sx={{ m: theme.spacing(6.4, 0) }}>
-          <AdditionalInformation optionText={optionText} />
+          {/* No severity information is associated with the total score */}
```

Comment:
I think its can be a relevant comment, but shall we remove or keep it ?

Reply:
It's intentional so I'd like to keep this one